tests/inference/models_predictions_tests/conftest.py

```diff
@@ -157,6 +163,31 @@ def yolov10_det_model() -> Generator[str, None, None]:
     shutil.rmtree(model_cache_dir)
 
 
+@pytest.fixture(scope="function")
+def sam2_tiny_model() -> Generator[str, None, None]:
+    model_id = "sam2/hiera_tiny"
+    model_cache_dir = fetch_and_place_model_in_cache(
+        model_id=model_id,
+        model_package_url="https://storage.googleapis.com/roboflow-tests-assets/sam2_tiny.zip",
+    )
+    yield model_id
+    shutil.rmtree(model_cache_dir)
+
+@pytest.fixture(scope="function")
+def sam2_small_model() -> Generator[str, None, None]:
+    model_id = "sam2/hiera_small"
+    model_cache_dir = fetch_and_place_model_in_cache(
+        model_id=model_id,
+        model_package_url="https://storage.googleapis.com/roboflow-tests-assets/sam2_small.zip",
+    )
+    yield model_id
+    shutil.rmtree(model_cache_dir)
+
+@pytest.fixture(scope="function")
+def sam2_small_truck_mask() -> Generator[np.ndarray, None, None]:
```

Comment:
If I understand correctly, this mask come from original SAM2 model, right?

Reply:
it's quite a big file to host in repo - could we assert either low res mask or just a sum of values?

Reply:
sum of values was kind of hard to guess good number of allowed error since the elements were unbounded.

Just tried serializing logits and got 257K file.  We could add these to test resources bucket instead (Peter actually suggested this while we were doing it, but I was being lazy and thought it wasn't that much bigger than an image😬), but main issue now would be we need to rewrite history to get large file out of git properly.